### PR TITLE
fix: prevent opening new tab when invoking function and show notification (#141)

### LINKED ISSUE #141: Avoid running invoke in terminal and show notification message
The invoke action opens up a new terminal tab to just show the message `function invoked` or an error. This can be displayed somewhere else so we can reduce the number of tabs opened

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #141 

image

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM but what is the reason the invoke output is not visible to the user. There are some cases where I think it might be usefull (async invocation where you need to know the execution id)

**Comment by lstocchi** [comment]
> LGTM but what is the reason the invoke output is not visible to the user. There are some cases where I think it might be usefull (async invocation where you need to know the execution id)

TBH i didn't think about the ID and you're right, it can be useful to know it. 

Initially I removed the tab because it just prints out the same things the user see/write in the wizard to invoke the function so it looked to me as a duplicate. But the ID is effectively not shown. Maybe showing the generated ID in the notification is enough? WDYT?

cc @mohitsuman

**Comment by jeffmaury** [comment]
Yes but does it requires to parse the non structured output because it is error prone. If we have JSON output to retrieve the ID then I'm +1

**Comment by lstocchi** [comment]
@jeffmaury latest cli 0.23.0 returns a JSON. I rebased this PR and used the newest cli. -> https://github.com/redhat-developer/intellij-knative/pull/142/commits/79d74ac9409d1bdf16d87e410d14dd093dd3ec9a

File: src/main/java/com/redhat/devtools/intellij/knative/actions/func/InvokeAction.java [hunk 1]
```diff
@@ -64,11 +68,21 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
         if (dialog.isOK()) {
             ExecHelper.submit(() -> {
                 try {
-                    knCli.invokeFunc(model);
+                    String id = knCli.invokeFunc(model);
+                    Notification notification = new Notification(NOTIFICATION_ID,
+                            "Invoked successfully",
+                            "Function " + name + " has been successfully invoked with execution id " + id + " !",
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/knative/actions/func/InvokeAction.java:74]: If by mistake id is not found in the JSON output, an empty string is returned thus we should handle this case to gererate proper information/notification to the user

**Review by jeffmaury — CHANGES_REQUESTED** [review]
LGTM

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**